docs/sp/alm.md

```diff
@@ -18,48 +20,36 @@ import "@pnp/sp/webs";
 const sp = spfi(...);
 
 // get the current context web's app catalog
-const catalog = await sp.web.getAppCatalog()();
-
-// you can also chain off the app catalog
-const apps = await sp.web.getAppCatalog()();
-console.log(apps);
+// this will be the site collection app catalog
+const availableApps = await sp.tenantAppcatalog();
 ```
 
+### Get site collection AppCatalog
+
 ```TypeScript
 import { spfi } from "@pnp/sp";
 import "@pnp/sp/appcatalog";
 import "@pnp/sp/webs";
 
 const sp = spfi(...);
 
-// you can get the tenant app catalog (or any app catalog) by using the getTenantAppCatalogWeb method
-const appCatWeb = await sp.getTenantAppCatalogWeb()();
-const appCatalog = await appCatWeb.getAppCatalog()();
-
-// you can get the tenant app catalog (or any app catalog) by passing in a url
-// get the tenant app catalog
-const tenantCatalog = await sp.web.getAppCatalog("https://mytenant.sharepoint.com/sites/appcatalog")();
-
-// get a different app catalog
-const catalog = await sp.web.getAppCatalog("https://mytenant.sharepoint.com/sites/anothersite")();
+// get the current context web's app catalog
+const availableApps = await sp.web.appcatalog();
 ```
 
-```TypeScript
-// alternatively you can create a new app catalog instance directly by importing the AppCatalog class
-import { IAppCatalog, AppCatalog } from '@pnp/sp/appcatalog';
+### Get site collection AppCatalog by URL
 
-const catalog: IAppCatalog = await AppCatalog("https://mytenant.sharepoint.com/sites/apps")();
-```
+If you know the url of the site collection whose app catalog you want you can use the following code. First you need to use one of the [methods to access a web](https://pnp.github.io/pnpjs/sp/webs/#access-a-web). Once you have the web instance you can call the `.appcatalog` property on that web instance.
+
+> If a given site collection does not have an app catalog trying to access it will throw an error.
 
 ```TypeScript
-// and finally you can combine use of the Web and AppCatalog classes to create an AppCatalog instance from an existing Web
 import { spfi } from "@pnp/sp";
 import { Web } from '@pnp/sp/webs';
-import { AppCatalog } from '@pnp/sp/appcatalog';
 
-const sp = spfi().using(...);
-const web = Web([sp.web, "https://mytenant.sharepoint.com/sites/apps"]);
-const catalog = await AppCatalog(web)();
+const sp = spfi(...);
+const web = Web([sp.web, "https://mytenant.sharepoint.com/sites/mysite"]);
+const catalog = await web.appcatalog();
```

Comment:
Should this be clarified?

web.appcatalog() would return apps, web.catalog would return the catalog.

const sp = spfi(...);
const web = Web([sp.web, "https://mytenant.sharepoint.com/sites/mysite"]);
const catalog = web.appcatalog;
const apps = await catalog();

Reply:
Not sure I follow. This is just assigning the `web.appcatalog` prop to `catalog`, then calling it. `catalog` can be `foo` But if its not clear we can update.